quaistats/quaistats.go

```diff
@@ -307,7 +307,7 @@ func (s *Service) loop(chainHeadCh chan core.ChainHeadEvent, txEventCh chan core
 				continue
 			}
 			// Keep sending status updates until the connection breaks
-			fullReport := time.NewTicker(15 * time.Second)
+			fullReport := time.NewTicker(5 * time.Second)
```

Comment:
We should probably set this back to 15, no?

Reply:
Done